### PR TITLE
Some cleaning with utils files

### PR DESCRIPTION
**Pull request by camillebrianceau**

### CODE REVIEW

File: clinicadl/commandline/modules_options/computational.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
     "this flag is already set to FSDP to that the zero flag is never actually removed.",
 )
 gpu = click.option(
-    "--no-gpu",
-    is_flag=True,
+    "--gpu/--no-gpu",
+    default=get_default("gpu", ComputationalConfig),
```
Comment (thibaultdvx) [inline clinicadl/commandline/modules_options/computational.py:23]: Seems to me that you made the opposite change in the previous PR (#629). Is it ok or is it a problem of rebasing?

File: tests/test_train_cnn.py [hunk 2]
```diff
@@ -13,10 +13,10 @@
...
     params=[
         "slice_cnn",
-        "image_cnn",
-        "patch_cnn",
-        "patch_multi_cnn",
-        "roi_cnn",
+        # "image_cnn",
+        # "patch_cnn",
+        # "patch_multi_cnn",
+        # "roi_cnn",
```
Comment (thibaultdvx) [inline tests/test_train_cnn.py]:
```suggestion
        "image_cnn",
        "patch_cnn",
        "patch_multi_cnn",
        "roi_cnn",
```

Is it an oversight?

**Review by thibaultdvx — COMMENTED** [review]
Just left two comments, LGTM otherwise :)

File: clinicadl/commandline/modules_options/computational.py (hunk 1 above)
Comment (camillebrianceau) [reply clinicadl/commandline/modules_options/computational.py:23]: Yes it was a mistake to do the opposite change !

**Pull request merged**